code/script.js

```diff
@@ -222,104 +227,189 @@ const generateBoard = () => {
   })
 }
 
-// Randomly select a person from the characters array and set as the value of the variable called secret
+// Function: setSecret randomly selects a person from the charachter array (this is the secret person).
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
+  console.log('secret person;', secret) //This console.log allows me to see who the secret person is for each game. 
 }
 
-// This function to start (and restart) the game
+// Function: starts the game. 
 const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
-  charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  charactersInPlay = CHARACTERS // Here we're setting charactersInPlay array to be all the characters to start with
 }
 
-// setting the currentQuestion object when you select something in the dropdown
+// Function: selectQuestion sets the currentQuestion when the user selects something in the dropdown (ie when the user formulates his/her question).
 const selectQuestion = () => {
-  const category = questions.options[questions.selectedIndex].parentNode.label
+  console.log('This is the question that I want answered')
+  const category = questions.options[questions.selectedIndex].parentNode.label // This variable stores what option group (category) the question belongs to.
+  const value = questions.value // This variable stores the actual value of the quesiton we've selected in the dropdown.
+    
+    if (category === 'hair') {
+      console.log('hair:', value)
+      currentQuestion = { 
+        category: category,
+        value: value
+      }
+    }
+  
+    else if (category === 'eyes') {
+      console.log('eyes:', value)
+      currentQuestion = { 
+        category: category,
+        value: value
+      }
+    }
 
-  // This variable stores what option group (category) the question belongs to.
-  // We also need a variable that stores the actual value of the question we've selected.
-  // const value =
+    else if (category === 'accessories') {
+      console.log('accessories:', value)
+      currentQuestion = { 
+        category: category,
+        value: value 
+      }
+    }
 
-  currentQuestion = {
-    category: category,
-    // value: value
+    else if (category === 'other') {
+      console.log('other:', value)
+      currentQuestion = {
+        category: category,
+        value: value 
+      }
+     }
```

Comment:
Here you could keep the original code, where the category and value for that current question is stored in the global variable (row 209) that is updated whenever the user asks a question:
`currentQuestion = {
    category: category,
    value: value
    }`
    This way all the if else statements could be deleted.